src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -58,9 +62,22 @@ function closeDrawer() {
 
 onMounted(() => {
   props.selectionEvent.subscribe((newVal) => {
+    origValue = JSON.parse(JSON.stringify(newVal));
     selectedComponent.value = newVal;
   });
 });
+
+function upsertAttribute(newAttr) {
+  if (!selectedComponent.value.attributes) selectedComponent.value.attributes = [];
```

Comment:
don't need that because component have to initialize themself its attribute.

Reply:
done